dask_sql/context.py

```diff
@@ -767,7 +766,8 @@ def _to_sql_string(self, s: "org.apache.calcite.sql.SqlNode", default_dialect=No
 
         try:
             return str(s.toSqlString(default_dialect))
-        except:  # pragma: no cover. Have not seen any instance so far, but better be safe than sorry.
+        # Have not seen any instance so far, but better be safe than sorry
+        except:  # noqa: E722; pragma: no cover
```

Comment:
This except block, along with handful others that should pop up in testing, is bare. Is there any way we can specify errors/exceptions for these? Or should we just `noqa` them?